### PR TITLE
Cross chain transactions quotes

### PR DESCRIPTION
**Pull request by riccardopillar**

### CODE REVIEW

**Review by kaushalrajbacancy — APPROVED** [review]

File: src/sdk/sdk.ts [hunk 1]
```diff
@@ -110,6 +110,7 @@ import {
   ExchangeService,
   CrossChainBridgeBuildTXResponse,
   CrossChainQuote,
+  MultiChainQuotes,
```
Comment (zincoshine) [inline src/sdk/sdk.ts:113]: What is the difference between CrossChainQuotes and MultiChainQuotes? Why do we need two separate data structures to achieve the same thing?

Can't we reuse the same structure?

**Review by zincoshine — CHANGES_REQUESTED** [review]

File: src/sdk/sdk.ts (hunk 1 above)
Comment (riccardopillar) [reply src/sdk/sdk.ts:113]: CrossChainQuote could include more fields from Lifi which cannot be shared with socketv2, socketv2 has its own data structure too

File: examples/src/18-multi-chain-quote.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  const fromAmount = utils.parseEther('0.000000000001'); // 0.1 USDC
+  const qouteRequestPayload = {
+    fromChainId: fromChainId,
+    toChainId: toChainId,
+    fromTokenAddress: fromTokenAddress,
+    toTokenAddress: toTokenAddress,
+    fromAmount: fromAmount,
+  };
+  console.log(qouteRequestPayload);
+  const quotes: BridgingQuotes = await sdk.getMultiChainQuotes(qouteRequestPayload);
```
Comment (zincoshine) [inline examples/src/18-multi-chain-quote.ts:58]: Can we please add the code to execute a selected quote as an example as well?

**Review by zincoshine — CHANGES_REQUESTED** [review]

**Review by zincoshine — APPROVED** [review]
LGTM

**Pull request merged**